Rubberduck.Main/Rubberduck.Main.csproj

```diff
@@ -235,6 +235,33 @@
   <ItemGroup>
     <Compile Include="App.cs" />
     <Compile Include="AppMenu.cs" />
+    <Compile Include="ComInteraction\UI\DockableWindowHost.cs" />
+    <Compile Include="ComInteraction\UnitTesting\AssertClass.cs" />
+    <Compile Include="ComInteraction\UnitTesting\FakesProvider.cs" />
+    <Compile Include="ComInteraction\UnitTesting\IAssert.cs" />
+    <Compile Include="ComInteraction\UnitTesting\IFake.cs" />
+    <Compile Include="ComInteraction\UnitTesting\IFakesProvider.cs" />
+    <Compile Include="ComInteraction\UnitTesting\IStub.cs" />
+    <Compile Include="ComInteraction\UnitTesting\IVerify.cs" />
+    <Compile Include="ComInteraction\UnitTesting\PermissiveAssertClass.cs" />
+    <Compile Include="ComUsers\UnitTesting\FakeBase.cs" />
```

Comment:
I'm not sure if I understand the distinction between `ComInteraction` and `ComUsers`... users interact, don't they?

Reply:
the naming there is somewhat up for debate... ComInteraction are classes that are ComVisible (excluding Extension). ComUsers are those that directly depend on those classes in ComInteraction. Maybe `ComDependents` could be a better name ...

Reply:
My suggestion would be to set up it in similar manner to what we are doing already with the `Rubberduck.VBEEditor`...

I would suggest that the root folder be `ComClientLibrary`, which I think is more descriptive of what it is for.

Dump all interfaces (and their associated folders) into `Abstract`
Dump all base classes into the root of `ComClientLibrary`
Dump all concrete implementations into corresponding folders (e.g. `Fakes` and `Stubs`)